src/mscorlib/src/System/Collections/Generic/EqualityComparer.cs

```diff
@@ -474,8 +438,7 @@ public override bool Equals(T x, T y)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I guess the attribute isn't needed anymore?Ü

Reply:
All other GetHashCode implementations in this file are marked with it, even if they are as simple as this one.